homeassistant/components/overkiz/cover_entities/awning.py

```diff
@@ -7,11 +7,11 @@
 
```

Comment:
> This package seems overkill for the current code.
> 
> _Originally posted by @MartinHjelmare in https://github.com/home-assistant/core/pull/64564#discussion_r793234898_

Would you prefer to have all cover entities in a single cover file? We originally split this up since we do the same for `water_heater` and `climate` (not in core yet). For example for climate we have twelve different implementations, and it is still growing.

Reply:
Yes, it's easier to read if it's all in the cover.py module. Only in very rare cases does it make it better to have a package.

Reply:
I thought it would be easier to read if the classes where split up, however I am fine to move them all back to cover.py. Will address this in this PR as well!

Reply:
Please do it separately since it's just a move.

Reply:
You are too fast! :D. I just realized the same and changed my reply. Thanks!